app/Http/Controllers/CommentsController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\CommentRequest; 
+use App\Models\Comment; 
+use Illuminate\Support\Facades\Auth;
+
+class CommentsController extends Controller
+{
+    public function store(CommentRequest $request,$id){
+   
+        $comment = new Comment();
+        $comment->user_id = Auth::id();
+        $comment->post_id=$request->input('post_id');
+        $comment->comment=$request->input('comment');
```

Comment:
細かいですが `=` の前後には空白を入れたほうが読みやすいのでおすすめです
```suggestion
        $comment->post_id = $request->input('post_id');
        $comment->comment = $request->input('comment');
```